src/components/shared/forms/pillFormControl.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Pill } from './pill';
+import './styles/pillFormControl.css';
+
+export const PillFormControl = ({ pills, onSvgClick, svg }) => (
```

Comment:
This seems like an odd name.  Perhaps PillList or PillGroup or something to indicate that multiple pills can/will be shown.  "form control" doesn't really help indicate that.